geom.js

```diff
@@ -1,19 +1,86 @@
-class Rectangle {
-  constructor(length, width) {
+// RUN IN REPL.IT TO CHECK
+class Rectangle{
+  constructor(length, width){
     this.length = length;
     this.width = width;
   }
+
+  area(){
+    return this.length * this.width;
+  }
+
+  isSquare(){
+    if (this.length == this.width) {
+      return "This is a square!"
+  } else {
+      return "This is just a rectangle; not a square."
+    }
+  }
+
+  perimeter(){
+    return "The perimeter is "(this.length * 2) + (this.width * 2)
+  }
```

Comment:
just return the integer. on the other hand, if you didn't have to do anything with the numbers returned by these functions, your JS Geometry code has better UX! 😆